massa-versioning-worker/src/versioning.rs

```diff
@@ -30,10 +30,8 @@ pub struct MipInfo {
     pub name: String,
     /// Network (or global) version (to be included in block header)
     pub version: u32,
-    /// Component concerned by this versioning (e.g. a new Block version)
-    pub component: MipComponent,
-    /// Component version
-    pub component_version: u32,
+    /// Components concerned by this versioning (e.g. a new Block version), and the associated component_version
+    pub components: HashMap<MipComponent, u32>,
```

Comment:
We probably can, but it does not make sense to have the same component updated twice in the same MIP. Instead of checking this manually, why not use the Hashmap features?